docs/src/main/asciidoc/panache-rest.adoc

```diff
@@ -0,0 +1,241 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
+////
+= Quarkus - Generating CRUD resources with Panache REST
+
+include::./attributes.adoc[]
+:extension-status: experimental
+
+A lot of web applications are monotonous CRUD applications with REST APIs that are tedious to write.
+To streamline this task, Panache REST can generate the basic CRUD endpoints for your entities and repositories.
+
+While this extension is still experimental and provides a limited feature set, we hope to get an early feedback for it.
+Currently this extension supports Hibernate ORM with Panache and can generate CRUD resources that work with `application/json` and `application/hal+json` content.
+
+include::./status-include.adoc[]
+
+== Setting up Panache REST
+
+To begin with:
+
+* Add the required dependencies to your `pom.xml`
+** Panache REST with Hiberante ORM extension (`quarkus-panache-rest-hibernate-orm`)
```

Comment:
Now that I read it like that, I've my doubts over the order, because originally it was supposed to be `X with Panache`, which means `X with style`.

This makes it looks like `Panache` is a noun and the subject instead of an adjective to make Hibernate ORM/REST stylish.

Perhaps it should be `Hibernate ORM REST with Panache` or `RESTful Hibernate ORM with Panache` or `REST for Hibernate ORM with Panache`?

Reply:
I guess, similarly, this guide (and the extension names) would better be "REST with Panache" and `rest-panache`?

Except this sort-of implies it applies to REST apps that aren't necessarily about data, and we plan to have a Panache layer for RESTEasy one day…

Perhaps `RESTful data with Panache`?

Reply:
Or `CRUD with Panache`?

Reply:
In any case, we need to make the extension name consistent with the naming rules we define here.

Reply:
Should I create a separate issue to agree on the name there?

Reply:
We can, yes, but this will heavily impact this guide.

Reply:
I could just update the guide together with the code once we agree on a name.

Reply:
OK fine.

Reply:
https://github.com/quarkusio/quarkus/issues/9293